src/platform/packages/shared/kbn-discover-utils/src/utils/sorting/get_sort_for_search_source.ts

```diff
@@ -7,16 +7,17 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import { type DataView, DataViewType } from '@kbn/data-views-plugin/common';
+import type { DataView, DataViewType } from '@kbn/data-views-plugin/common';
 import type { EsQuerySortValue, SortDirection } from '@kbn/data-plugin/common';
-import type { SortOrder } from '@kbn/saved-search-plugin/public';
-import { getSort } from './get_sort';
+import { type SortOrder, getSort } from './get_sort';
 import {
   getESQuerySortForTimeField,
   getESQuerySortForTieBreaker,
   DEFAULT_TIE_BREAKER_NAME,
 } from './get_es_query_sort';
 
+const DATA_VIEW_TYPE_ROLLUP: `${DataViewType.ROLLUP}` = 'rollup';
```

Comment:
`kbn-discover-utils` can't access non-types from `data-views-plugin`, and I didn't want to create a separate `data-views-utils` package for this PR, so this is a type safe workaround for now.